serverless/functions/serverless-functions-annotations.adoc

```diff
@@ -0,0 +1,19 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-functions-attributes"]
+= Adding annotations to functions
+include::modules/common-attributes.adoc[]
+:context: serverless-functions-annotations
+
+toc::[]
+
+You can add annotations to serverless functions using the `func.yaml` configuration file.
+
+[IMPORTANT]
+====
+There are two limitations of the function annotation feature:
+
+* Once a function annotation propagates to the corresponding service on the cluster, it cannot be removed from the service by deleting it from the `func.yaml` file.
```

Comment:
Tell users how to delete it? Do they have to edit the YAML for the service and manually delete it that way?

Also, sort of nitpicking, but when we say service throughout these docs, do we mean Knative service, and if so can you call that out, since it could be any k8s service otherwise?

Reply:
Yes - they would need to delete it via the Developer Console or manually editing the YAML. For now, anyway.

Reply:
Added this information right in this paragraph. I can also create a full separate procedure doc for that, should I?

Reply:
Nah I think it's OK just to point it out